### PR TITLE
Elisa Levet

### PR DESCRIPTION
**Pull request by elisalevet**
Hii!
This is my Readme

### CODE REVIEW

**Comment by apaniagua6** [comment]
Taking a look..

File: Elisa/readme.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+  return s === newString
+}
+```
+Optimized
+```Javascript
+const isPalindrome =(string) => {
+    string= string.replace(/[^a-zA-Z0-9]/g,"").toLowerCase();
+    let leftIdx = 0;
+    let rightIdx = string.length - 1;
```
Comment (apaniagua6) [inline Elisa/readme.md:153]: Good job here. I recommend you keep a sticky with notes for this type of syntax on your desktop. Anything that requires more than 10 seconds to think about, just write a sticky for it.

File: Elisa/readme.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  let result = []
+	for (let i = 0; i < nums.length; i++) {
+		for (let j = i + 1; j < nums.length; j++) {
+			if (nums[i] + nums[j] === target) {
+				result.push(i,j);
+			}
+		}
+	}
+	return result;
+}
```
Comment (apaniagua6) [inline Elisa/readme.md:18]: Please be consistent with your ending  ; in javascript. Good job in adding the brute force and optimal solution. Please include the Big O notation of each approach so you  are quick with these space and time complexities.

File: Elisa/readme.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    }
+    return true
+}
+```
+## Question #6b Almost Palindrome (Easy)
+```Javascript
+const almostPalindrome = (string) => {
+
+}
+```
```
Comment (apaniagua6) [inline Elisa/readme.md:170]: Almost there. Please finish the solution.

**Review by sphinx20 — APPROVED** [review]

**Pull request merged**